src/OpenTelemetry.Exporter.Geneva/External/LinuxTracepoints-Net/Provider/EventHeaderValue128.cs

```diff
@@ -0,0 +1,50 @@
+#nullable enable
+
+namespace Microsoft.LinuxTracepoints.Provider
+{
+    using System;
+    using System.Runtime.CompilerServices;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// 16-byte raw value storage for IPv6, UUID, or any other 16-byte value
+    /// that needs to be used as a field in an EventHeader event.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct EventHeaderValue128
```

Comment:
Is this by mistake or intentionally kept public here?

Reply:
It was a mistake. I have updated it to internal.